publish.go

```diff
@@ -16,13 +16,14 @@ import (
 // EventMetadata represents the meta information which comes along with all Nakadi events. For publishing
 // purposes only the fields eid and occurred_at must be present.
 type EventMetadata struct {
-	EID        string     `json:"eid"`
-	OccurredAt time.Time  `json:"occurred_at"`
-	EventType  string     `json:"event_type,omitempty"`
-	Partition  string     `json:"partition,omitempty"`
-	ParentEIDs []string   `json:"parent_eids,omitempty"`
-	FlowID     string     `json:"flow_id,omitempty"`
-	ReceivedAt *time.Time `json:"received_at,omitempty"`
+	EID        string      `json:"eid"`
+	OccurredAt time.Time   `json:"occurred_at"`
+	EventType  string      `json:"event_type,omitempty"`
+	Partition  string      `json:"partition,omitempty"`
+	ParentEIDs []string    `json:"parent_eids,omitempty"`
+	FlowID     string      `json:"flow_id,omitempty"`
+	ReceivedAt *time.Time  `json:"received_at,omitempty"`
+	SpanCTX    interface{} `json:"span_ctx,omitempty"`
```

Comment:
isn't its a more common to write `Ctx` instead of `CTX`?
```suggestion
	SpanCtx    interface{} `json:"span_ctx,omitempty"`
```